site/content/features/stringtemplates.md

```diff
@@ -1,12 +1,14 @@
 ---
-title: String Templates (JEP 430)
+title: String Templates (JEP 430, 459, 465, withdrawn)
 copyright: Cay S. Horstmann 2023. All rights reserved.
-jep: 430
-jdkversion: 21
+jep: 430, 459, 465
+jdkversion: 21, 22
```

Comment:
Only a single Java version allowed here, otherwise there will be a broken link to JDK "21, 22".